doc/installation/index.rst

```diff
@@ -28,6 +28,47 @@ compile NEST from source, see section** :ref:`advanced_install`.
 
            sudo apt-get install nest
 
+
+   .. tab:: Debian
+
+       Debian users can install NEST via the Ubuntu PPA repository.
+
+       1. Create a new ``apt`` repository entry in ``/etc/apt/sources.list.d/nest-simulator-ubuntu-nest-XXX.list`` by:
+
+       .. code-block:: bash
+
+          sudo apt install devscripts build-essential software-properties-common dpkg-dev
+          sudo add-apt-repository --enable-source ppa:nest-simulator/nest
+
+       2. Disable the binary package in the repository file by commenting it out.
+
+       3. Import the PPA GPC key and rebuild the package:
+
+       .. code-block:: bash
+
+          sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
+                           --recv-keys 0CF7539642ABD23CBCA8D487F0B8B6C5EC02D7DD
+          sudo apt update
+          sudo apt source --build nest
+
+       4. Install any missing dependencies, if ``apt`` tells you so.
+
+       5. After installing the dependencies, enter ``sudo apt source --build nest`` again.
```

Comment:
There seems to be something additionally missing for Perl in this step, when trying this guide on Debian 11
```bash
# sudo apt source --build nest
Reading package lists... Done
Skipping already downloaded file 'nest_2.20.0-0~202001311135~ubuntu20.04.1.tar.xz'
Skipping already downloaded file 'nest_2.20.0-0~202001311135~ubuntu20.04.1.dsc'
Need to get 0 B of source archives.
Skipping unpack of already unpacked source in nest-2.20.0
dpkg-buildpackage: info: source package nest
dpkg-buildpackage: info: source version 2.20.0-0~202001311135~ubuntu20.04.1
dpkg-buildpackage: info: source distribution focal
dpkg-buildpackage: info: source changed by Steffen Graber <s.graber@fz-juelich.de>
dpkg-buildpackage: info: host architecture amd64
 dpkg-source --before-build .
 debian/rules clean
LC_ALL=en_US.UTF-8 dh clean --with python3 --builddirectory=build
dh: error: unable to load addon python3: Can't locate Debian/Debhelper/Sequence/python3.pm in @INC (you may need to install the Debian::Debhelper::Sequence::python3 module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at (eval 15) line 1.
BEGIN failed--compilation aborted at (eval 15) line 1.

make: *** [debian/rules:16: clean] Error 255
dpkg-buildpackage: error: debian/rules clean subprocess returned exit status 2
E: Build command 'cd nest-2.20.0 && dpkg-buildpackage -b -uc' failed.
```

Reply:
@terhorstd Could you please install

     sudo apt install python3-all dh-python

and then again

    sudo apt source --build nest
    sudo dpkg --install nest-simulator-x.y.z~NUMBER~ubuntu20.04.1_amd64.deb

Reply:
build is now successful, though it finishes with a warning (can be handled in different PR)
```
[...]
-- Installing: /root/nest-2.20.0/debian/nest/usr/share/doc/nest/examples/pynest/twoneurons.py
-- Installing: /root/nest-2.20.0/debian/nest/usr/share/doc/nest/examples/pynest/vinit_example.py
make[2]: Leaving directory '/root/nest-2.20.0'
# Now changing nest_vars.sh setting the correct paths
sed -i 's/\/build\/\(.*\)nest\(.*\)\/debian\/nest//g' debian/nest/usr/bin/nest_vars.sh
make[1]: Leaving directory '/root/nest-2.20.0'
   dh_installchangelogs -O--builddirectory=build
   dh_systemd_enable -O--builddirectory=build
   dh_python3 -O--builddirectory=build
E: dh_python3 dh_python3:176: no package to act on (python3-foo or one with ${python3:Depends} in Depends)
   dh_installinit -O--builddirectory=build
   dh_systemd_start -O--builddirectory=build
   dh_perl -O--builddirectory=build
   dh_link -O--builddirectory=build
   dh_strip_nondeterminism -O--builddirectory=build
   dh_compress -O--builddirectory=build
   dh_fixperms -O--builddirectory=build
   dh_missing -O--builddirectory=build
   dh_strip -O--builddirectory=build
   dh_makeshlibs -O--builddirectory=build
   dh_installdeb -O--builddirectory=build
   dh_gencontrol -O--builddirectory=build
dpkg-gencontrol: warning: Depends field of package nest: substitution variable ${shlibs:Depends} used, but is not defined
dpkg-gencontrol: warning: Depends field of package nest: substitution variable ${shlibs:Depends} used, but is not defined
   dh_md5sums -O--builddirectory=build
   dh_builddeb -O--builddirectory=build
dpkg-deb: building package 'nest-dbgsym' in '../nest-dbgsym_2.20.0-0~202001311135~ubuntu20.04.1_amd64.deb'.
dpkg-deb: building package 'nest' in '../nest_2.20.0-0~202001311135~ubuntu20.04.1_amd64.deb'.
#dh binary
 dpkg-genbuildinfo --build=binary
 dpkg-genchanges --build=binary >../nest_2.20.0-0~202001311135~ubuntu20.04.1_amd64.changes
dpkg-genchanges: info: binary-only upload (no source code included)
 dpkg-source --after-build .
dpkg-buildpackage: info: binary-only upload (no source included)

```

Reply:
```suggestion
       5. After installing the dependencies, enter ``sudo apt source --build nest`` again.
       When the build finished, look for lines like
       ```
      dpkg-deb: building package 'nest-dbgsym' in '../nest-dbgsym_2.20.0-0~202001311135~ubuntu20.04.1_amd64.deb'.
      dpkg-deb: building package 'nest' in '../nest_2.20.0-0~202001311135~ubuntu20.04.1_amd64.deb'.
      #dh binary
       dpkg-genbuildinfo --build=binary
       dpkg-genchanges --build=binary >../nest_2.20.0-0~202001311135~ubuntu20.04.1_amd64.changes
      ```
      and note down the full package name. In the above example this would be 
      `nest_2.20.0-0~202001311135~ubuntu20.04.1_amd64.deb`, where the
       number `202001311135` and potentially the Ubuntu version number may
      be different.
```